projects/packages/search/src/instant-search/components/overlay.scss

```diff
@@ -42,7 +42,35 @@ $overlay-horizontal-padding-lg: 3em;
 		padding: $overlay-vertical-padding-lg $overlay-horizontal-padding-lg;
 	}
 
-	h1, h2, h3, h4, h5, h6 {
+	// TODO: look into a more sustainable approach to reduce the number of !important being used to override the button styles
+	.jetpack-instant-search__search-sort-option,
```

Comment:
I think these belong in their respective SCSS files rather than in overlay.scss. For example, the sort option style should be in `projects/packages/search/src/instant-search/components/search-sort.scss`.

Maybe we do want to add the `!important` to the reset button styles mixin after all?